test/functional/apps/home/_navigation.js

```diff
@@ -81,20 +90,18 @@ export default function ({ getService, getPageObjects }) {
     });
 
     it('encodes portions of the URL as necessary', async () => {
-      const host = process.env.TEST_KIBANA_HOST || 'localhost';
-      const port = process.env.TEST_KIBANA_PORT || '5620';
-      const basePath = `http://${host}:${port}`;
-
+      const basePath = getBasePath();
```

Comment:
After this line, we need to have the following to log in: 
await PageObjects.common.navigateToApp('home');

Reply:
done